src/java/org/apache/nutch/parse/ParseSegment.java

```diff
@@ -56,6 +57,14 @@
   private ParseUtil parseUtil;
 
   private boolean skipTruncated;
+  
+  public static final String PARSER_MODELFILTER="parser.modelfilter";
```

Comment:
I would argue that the key's are unsuitable. As this is meant to be an URLFilter, the keys should refelect that... not parser related but instead URLFilter specific